lib/rebaser-server/src/config.rs

```diff
@@ -66,7 +66,7 @@ pub struct Config {
     #[builder(default = "random_instance_id()")]
     instance_id: String,
 
-    #[builder(default = "5000")]
+    #[builder(default = "800")]
```

Comment:
@fnichol can you think of any issues with the leader election stuff that changing this timer might cause @jkeiser